src/database/models/index.js

```diff
@@ -21,6 +21,7 @@ fs
   .filter((file) => (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js'))
   .forEach((file) => {
     // eslint-disable-next-line global-require
+    // eslint-disable-next-line import/no-dynamic-require
```

Comment:
same here

Reply:
You must write unit tests for your functions

Reply:
the unit tests have been added